webapi/Controllers/ChatHistoryController.cs

```diff
@@ -260,4 +268,108 @@ public async Task<IActionResult> GetAllChatSessionsAsync(string userId)
 
         return this.NotFound($"No chat session found for chat id '{chatId}'.");
     }
+
+    /// <summary>
+    /// Delete a chat session.
+    /// </summary>
+    /// <param name="userId">Unique Id of user who initiated the request.</param>
+    /// <param name="sessionId">The chat id.</param>
+    [HttpDelete]
+    [Route("chatSession/{sessionId:guid}")]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> DeleteChatSessionAsync(
+        [FromServices] IHubContext<MessageRelayHub> messageRelayHubContext,
+        [FromServices] IAuthInfo authInfo,
```

Comment:
This was already passed into the constructor and stored as a member variable, so you don't need to pass it in here.